samples/ai-image-tagging/src/components/ImageTagging.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, CheckboxField } from '@contentful/forma-36-react-components';
+
+import { mergeTags, requestTags } from '../lib/tags';
+
+import './ImageTagging.css';
+
+export class ImageTagging extends React.Component {
+  static propTypes = {
+    sdk: PropTypes.object.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      overwriteTags: true,
+    }
+  }
+
+  componentDidMount() {
+    this.props.sdk.window.startAutoResizer();
```

Comment:
As far as I see `this.props.sdk.window.startAutoResizer();` is already called once in `src/index.js`.